mapper/mapper.go

```diff
@@ -124,10 +124,16 @@ func mapEmergencyBanner(bannerData zebedee.EmergencyBanner) coreModel.EmergencyB
 
 func createPreGTMJavaScript(title string, description model.ReleaseDescription) []template.JS {
 	releaseStatus := "cancelled"
+	censusTag := ""
```

Comment:
This is a valid method to instantiate an empty string variable but this is a bit cleaner 
```suggestion
	var censusTag string
```

Reply:
Thank you for this comment. This is now done.